python/cudf_polars/cudf_polars/experimental/parallel.py

```diff
@@ -225,17 +226,23 @@ def _(
     # Generate pointwise (embarrassingly-parallel) tasks by default
     child_names = [get_key_name(c) for c in ir.children]
     bcast_child = [partition_info[c].count == 1 for c in ir.children]
-    return {
-        key: (
-            ir.do_evaluate,
+    tasks = {}
```

Comment:
Sorry to keep commenting on the same chunk of code, but I'm still not sure I understand why dict comprehension needs to go away?